helm/values.yaml

```diff
@@ -704,10 +704,10 @@ gateway:
       - am.example.com
     annotations:
       kubernetes.io/ingress.class: nginx
-      nginx.ingress.kubernetes.io/ssl-redirect: "false"
```

Comment:
why is it removed instead of setting it as true?

Reply:
As mentioned in the PR comment, if we force it to true, then if you want to use HTTP without TLS, it will fail.
The proposal here is to keep default config to let both case: with or without TLS works.